### PR TITLE
Update failed_when pattern for cargo verification step

### LINKED ISSUE #21: overwrite message has changed for cargo
"use --force to override" now replaces the "Add --force to overwrite" in `tasks/cargo.yml`

(may depends of the cargo version)

Fix in `task/cargo.yml`:
```
      failed_when:
        - ("use --force to override" not in item.stderr) and
          ("Finished" not in item.stderr)
```

### PR DESCRIPTION
**Pull request by saulrh**
Fixes #21

### CODE REVIEW

**Comment by RobQuistNL** [comment]
It would be nice if this can be merged :)

File: tasks/cargo.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   with_items: "{{ cargo_result.results }}"
   changed_when: false
   failed_when:
-    - ("Add --force to overwrite" not in item.stderr) and
+    - ("use --force to override" not in item.stderr) and
```
Comment (RobQuistNL) [inline tasks/cargo.yml:15]: Maybe better to replace this with `(("use --force to override" not in item.stderr) or ("Add --force to overwrite" not in item.stderr)) and` so it stays backwards compatible

**Comment by RobQuistNL** [comment]
@fubarhouse anything I can do to get this through? (or perhaps know a workaround?) :) This is the only remaining failing task in my provisioning script

**Comment by fubarhouse** [comment]
I'll put this on today's plate 🙂

**Comment by RobQuistNL** [comment]
<3 awesome, thanks

**Comment by fubarhouse** [comment]
I've tested it with my role tester - I'll update the test suite and get a release done _soon_.
Can't do that until after business hours - it would take more than a couple of minutes to do and I am at work.

Test is passing, so I'll merge - thanks for contributing!

**Comment by RobQuistNL** [comment]
Awesome! Thanks a ton. I've just tried removing and installing again through galaxy, but it looks like that doesn't download the latest version.

I also would rather not use galaxy (because I'd like to have everything in source control) - is there a way for me to, for example, include this project as a git module?

**Comment by RobQuistNL** [comment]
EDIT: It does download the latest version (thats on galaxy :+) ) but not the master branch which includes this fix.

**Pull request merged**